basic/06b_kinematics_fwd_kinematics.cpp

```diff
@@ -30,19 +34,28 @@ int main()
   }
 
   // Add a callback function to print (x,y,z) position
-  Eigen::Matrix4d transform;
-  group->addFeedbackHandler([&model, &transform](const GroupFeedback& group_fbk)
-  {
-    Eigen::VectorXd angles = group_fbk.getPosition();
-    model->getEndEffector(angles, transform);
-    std::cout << std::setw(0.2) <<
-       "x " << transform(0,3) <<
-      " y " << transform(1,3) <<
-      " z " << transform(2,3) << std::endl;
-  });
+  hebi::robot_model::Matrix4dVector transforms;
+  GroupFeedback group_fbk(group->size());
+  for(size_t i = 0; i < 50; i++) {
+    if (group->getNextFeedback(group_fbk)){
+      Eigen::VectorXd angles = group_fbk.getPosition();
+      model->getFK(HebiFrameTypeOutput, angles, transforms);
+
+      transforms.emplace(transforms.begin(),Eigen::Matrix<double,4,4>::Identity());
+      std::vector<std::vector<double>> lines_x;
+      std::vector<std::vector<double>> lines_y;
+      std::vector<std::vector<double>> lines_z;
+
+      plt::clf();
+      for(size_t j = 0; j < transforms.size(); ++j) {
+	plot_3dtriad(transforms[j],&lines_x,&lines_y,&lines_z, static_cast<bool>(j));
+      }
+      plt::pause(1);
+    }
+  }
   
   // Control the robot at 100 Hz for 30 seconds
-  std::this_thread::sleep_for(std::chrono::seconds(30));
+  //std::this_thread::sleep_for(std::chrono::seconds(30));
```

Comment:
I like switching this to the "getNextFeedback" approach; we should remove this comment and the next line, which isn't needed anymore.